tests/Properties.hs

```diff
@@ -93,6 +96,23 @@ prop_lines_lazy2 =
 
 prop_strip x = C.strip x == (C.dropSpace . C.reverse . C.dropSpace . C.reverse) x
 
+prop_toConstr :: P.ByteString -> Property
+prop_toConstr bs = True ==> "pack" == ((showConstr  . toConstr) bs)
```

Comment:
The `True ==>` bit seems unnecessary.

```suggestion
prop_toConstr bs = "pack" == ((showConstr  . toConstr) bs)
```

(Dito for `prop_gshow_equal` and  `prop_gshow_string`)